src/tests/multihost/ad/test_dyndns.py

```diff
@@ -0,0 +1,536 @@
+"""Automation tests for dynamic DNS
+
+:requirement: dyndns
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
++:status: approved
+"""
+
+import pytest
+import time
+import random
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.utils import ADDNS
+
+
+@pytest.fixture(scope="class")
+def change_client_hostname(session_multihost, request):
+    """ Change client hostname to a truncated version in the AD domain"""
```

Comment:
Is this supposed to be truncated? it looks fully qualified here:
~~~
sssd.testlib.common.qe_class.QeHost.hostname1.cmd29 - DEBUG - RUN hostnamectl set-hostname "client44.domain-0men.com"
~~~